src/backend/parser/cypher_gram.y

```diff
@@ -2415,7 +2434,7 @@ static cypher_relationship *build_VLE_relation(List *left_arg,
         (cnl->name == NULL && cnr->label != NULL) ||
         (cnl->name == NULL && cnr->props != NULL))
     {
-        cnl->name = create_unique_name("_vle_function_start_var");
+        cnl->name = create_unique_name(AGE_DEFAULT_PREFIX"vle_function_start_var");
```

Comment:
Should there be a `_` before `vle_function_start_var`, because `AGE_DEFAULT_PREFIX` does not end with a `_`? Just for readability.

Also appear in line 2469.

Reply:
AGE_DEFAULT_PREFIX does end in an _ -

```
/*
 * Every internal alias or variable name should be prefixed
 * with AGE_DEFAULT_PREFIX. Grammer restricts variables
 * prefixed with _age_default_ in user query to be used.
 */
#define AGE_DEFAULT_PREFIX "_age_default_"
#define AGE_DEFAULT_ALIAS_PREFIX AGE_DEFAULT_PREFIX"alias_"
#define AGE_DEFAULT_VARNAME_PREFIX AGE_DEFAULT_PREFIX"varname_"
```

Reply:
Oh. I may have looked at the wrong macro. My apologies.